<!DOCTYPE HTML>
<html lang="en-US" manifest="../manifest.appcache">
    
    <head>
        
        <meta charset="UTF-8">
        <title>FAQ | GenomeRunner</title>
        <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
        <meta name="description" content="">
        <meta name="generator" content="GitBook 0.6.0">
        <meta name="HandheldFriendly" content="true"/>
        <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
        <meta name="apple-mobile-web-app-capable" content="yes">
        <meta name="apple-mobile-web-app-status-bar-style" content="black">
        <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">
        
    
    
    <meta name="author" content="mdozmorov">
    
    
    <link rel="next" href="../misc/definitions.html" />
    
    
    <link rel="prev" href="../index.html" />
    

        
    </head>
    <body>
        
        
<link rel="stylesheet" href="../gitbook/style.css">


        
    <div class="book" data-github="mdozmorov/grdocs" data-level="1" data-basepath=".." data-revision="1405448325392">
    <div class="book-header">
    <!-- Actions Left -->
    <a href="#" class="btn pull-left toggle-summary" aria-label="Toggle summary"><i class="fa fa-align-justify"></i></a>
    
    <a href="https://github.com/mdozmorov/grdocs" target="_blank" class="btn pull-left home-bookmark" aria-label="GitHub home"><i class="fa fa-bookmark-o"></i></a>
    
    <a href="#" class="btn pull-left toggle-search" aria-label="Toggle search"><i class="fa fa-search"></i></a>
    <span id="font-settings-wrapper">
        <a href="#" class="btn pull-left toggle-font-settings" aria-label="Toggle font settings"><i class="fa fa-font"></i>
        </a>
        <div class="dropdown-menu font-settings">
    <div class="dropdown-caret">
        <span class="caret-outer"></span>
        <span class="caret-inner"></span>
    </div>

    <div class="btn-group btn-block">
        <button id="reduce-font-size" class="btn btn-default">A</button>
        <button id="enlarge-font-size" class="btn btn-default">A</button>
    </div>

    <ul class="list-group font-family-list">
        <li class="list-group-item" data-font="0">Serif</li>
        <li class="list-group-item" data-font="1">Sans</li>
    </ul>

    <div class="btn-group btn-group-xs btn-block color-theme-list">
        <button type="button" class="btn btn-default" id="color-theme-preview-0" data-theme="0">White</button>
        <button type="button" class="btn btn-default" id="color-theme-preview-1" data-theme="1">Sepia</button>
        <button type="button" class="btn btn-default" id="color-theme-preview-2" data-theme="2">Night</button>
    </div>
</div>

    </span>

    <!-- Actions Right -->
    
    <a href="#" target="_blank" class="btn pull-right google-plus-sharing-link sharing-link" data-sharing="google-plus" aria-label="Share on Google Plus"><i class="fa fa-google-plus"></i></a>
    
    
    <a href="#" target="_blank" class="btn pull-right facebook-sharing-link sharing-link" data-sharing="facebook" aria-label="Share on Facebook"><i class="fa fa-facebook"></i></a>
    
    
    <a href="#" target="_blank" class="btn pull-right twitter-sharing-link sharing-link" data-sharing="twitter" aria-label="Share on Twitter"><i class="fa fa-twitter"></i></a>
    

    
    <a href="https://github.com/mdozmorov/grdocs/stargazers" target="_blank" class="btn pull-right count-star hidden-xs"><i class="fa fa-star-o"></i> Star (<span>-</span>)</a>
    <a href="https://github.com/mdozmorov/grdocs/watchers" target="_blank" class="btn pull-right count-watch hidden-xs"><i class="fa fa-eye"></i> Watch (<span>-</span>)</a>
    

    <!-- Title -->
    <h1>
        <i class="fa fa-spinner fa-spin"></i>
        <a href="../" >GenomeRunner</a>
    </h1>
</div>

    

<div class="book-summary">
    <div class="book-search">
        <input type="text" placeholder="Search" class="form-control" />
    </div>
    <ul class="summary">
        
        
        
        <li>
            <a href="https://github.com/mdozmorov" target="blank" class="author-link">About the author</a>
        </li>
        

        
        
        <li>
            <a href="https://github.com/mdozmorov/grdocs/issues" target="blank"class="issues-link">Questions and Issues</a>
        </li>
        

        
        
        <li>
            <a href="https://github.com/mdozmorov/grdocs/edit/master/misc/Faq.md" target="blank" class="contribute-link">Edit and Contribute</a>
        </li>
        

	

        
        <li class="divider"></li>
        

        
    
        
        <li class="chapter " data-level="0" data-path="index.html">
            
                
                    <a href="../index.html">
                        <i class="fa fa-check"></i>
                        
                         Introduction
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter active" data-level="1" data-path="misc/Faq.html">
            
                
                    <a href="../misc/Faq.html">
                        <i class="fa fa-check"></i>
                        
                            <b>1.</b>
                        
                         FAQ
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="2" data-path="misc/definitions.html">
            
                
                    <a href="../misc/definitions.html">
                        <i class="fa fa-check"></i>
                        
                            <b>2.</b>
                        
                         Definitions
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="3" data-path="misc/newsAndUpdates.html">
            
                
                    <a href="../misc/newsAndUpdates.html">
                        <i class="fa fa-check"></i>
                        
                            <b>3.</b>
                        
                         News and Updates
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="4" data-path="installation/installation.html">
            
                
                    <a href="../installation/installation.html">
                        <i class="fa fa-check"></i>
                        
                            <b>4.</b>
                        
                         Installation
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="5" data-path="installation/uninstallation.html">
            
                
                    <a href="../installation/uninstallation.html">
                        <i class="fa fa-check"></i>
                        
                            <b>5.</b>
                        
                         Uninstalling
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6" data-path="installation/modules.html">
            
                
                    <a href="../installation/modules.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.</b>
                        
                         Modules
                    </a>
                
            
            
            <ul class="articles">
                
    
        
        <li class="chapter " data-level="6.1" data-path="dbcreator/dbcreator.html">
            
                
                    <a href="../dbcreator/dbcreator.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.1.</b>
                        
                         Dbcreator module
                    </a>
                
            
            
            <ul class="articles">
                
    
        
        <li class="chapter " data-level="6.1.1" data-path="dbcreator/dbcreator_ucsc.html">
            
                
                    <a href="../dbcreator/dbcreator_ucsc.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.1.1.</b>
                        
                         dbcreator_ucsc
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6.1.2" data-path="dbcreator/dbcreator_encodeDCC.html">
            
                
                    <a href="../dbcreator/dbcreator_encodeDCC.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.1.2.</b>
                        
                         dbcreator_encodeDCC
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6.1.3" data-path="dbcreator/dbcreatorWorks.html">
            
                
                    <a href="../dbcreator/dbcreatorWorks.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.1.3.</b>
                        
                         How dbcreator works
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6.1.4" data-path="dbcreator/dbcreatorFiltering.html">
            
                
                    <a href="../dbcreator/dbcreatorFiltering.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.1.4.</b>
                        
                         Filtering
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6.1.5" data-path="dbcreator/dbcreatorStructure.html">
            
                
                    <a href="../dbcreator/dbcreatorStructure.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.1.5.</b>
                        
                         Database structure
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6.1.6" data-path="dbcreator/dbcreatorBackground.html">
            
                
                    <a href="../dbcreator/dbcreatorBackground.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.1.6.</b>
                        
                         Background
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6.1.7" data-path="dbcreator/dbcreatorCustom.html">
            
                
                    <a href="../dbcreator/dbcreatorCustom.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.1.7.</b>
                        
                         Custom data
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6.1.8" data-path="dbcreator/dbcreatorFAQ.html">
            
                
                    <a href="../dbcreator/dbcreatorFAQ.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.1.8.</b>
                        
                         Dbcreator FAQ
                    </a>
                
            
            
        </li>
    

            </ul>
            
        </li>
    
        
        <li class="chapter " data-level="6.2" data-path="optimizer/optimizer.html">
            
                
                    <a href="../optimizer/optimizer.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.2.</b>
                        
                         Optimizer module
                    </a>
                
            
            
            <ul class="articles">
                
    
        
        <li class="chapter " data-level="6.2.1" data-path="optimizer/optimizerParallel.html">
            
                
                    <a href="../optimizer/optimizerParallel.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.2.1.</b>
                        
                         Parallelization
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6.2.2" data-path="optimizer/optimizerFAQ.html">
            
                
                    <a href="../optimizer/optimizerFAQ.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.2.2.</b>
                        
                         Optimizer FAQ
                    </a>
                
            
            
        </li>
    

            </ul>
            
        </li>
    
        
        <li class="chapter " data-level="6.3" data-path="server/server.html">
            
                
                    <a href="../server/server.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.3.</b>
                        
                         Server module
                    </a>
                
            
            
            <ul class="articles">
                
    
        
        <li class="chapter " data-level="6.3.1" data-path="server/serverParallel.html">
            
                
                    <a href="../server/serverParallel.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.3.1.</b>
                        
                         Parallelization
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="6.3.2" data-path="server/serverFAQ.html">
            
                
                    <a href="../server/serverFAQ.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.3.2.</b>
                        
                         Server FAQ
                    </a>
                
            
            
        </li>
    

            </ul>
            
        </li>
    
        
        <li class="chapter " data-level="6.4" data-path="hypergeom4/hypergeom4.html">
            
                
                    <a href="../hypergeom4/hypergeom4.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.4.</b>
                        
                         Command line module
                    </a>
                
            
            
            <ul class="articles">
                
    
        
        <li class="chapter " data-level="6.4.1" data-path="hypergeom4/hypergeom4Faq.html">
            
                
                    <a href="../hypergeom4/hypergeom4Faq.html">
                        <i class="fa fa-check"></i>
                        
                            <b>6.4.1.</b>
                        
                         Command line FAQ
                    </a>
                
            
            
        </li>
    

            </ul>
            
        </li>
    

            </ul>
            
        </li>
    
        
        <li class="chapter " data-level="7" data-path="hypergeom4/results.html">
            
                
                    <a href="../hypergeom4/results.html">
                        <i class="fa fa-check"></i>
                        
                            <b>7.</b>
                        
                         Results
                    </a>
                
            
            
            <ul class="articles">
                
    
        
        <li class="chapter " data-level="7.1" data-path="hypergeom4/enrichment.html">
            
                
                    <a href="../hypergeom4/enrichment.html">
                        <i class="fa fa-check"></i>
                        
                            <b>7.1.</b>
                        
                         Enrichment results
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="7.2" data-path="hypergeom4/episimilarity.html">
            
                
                    <a href="../hypergeom4/episimilarity.html">
                        <i class="fa fa-check"></i>
                        
                            <b>7.2.</b>
                        
                         Epigenomic similarity results
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="7.3" data-path="hypergeom4/annotation.html">
            
                
                    <a href="../hypergeom4/annotation.html">
                        <i class="fa fa-check"></i>
                        
                            <b>7.3.</b>
                        
                         Annotation results
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="7.4" data-path="hypergeom4/logs.html">
            
                
                    <a href="../hypergeom4/logs.html">
                        <i class="fa fa-check"></i>
                        
                            <b>7.4.</b>
                        
                         Logs and debug info
                    </a>
                
            
            
        </li>
    
        
        <li class="chapter " data-level="7.5" data-path="hypergeom4/resultsFaq.html">
            
                
                    <a href="../hypergeom4/resultsFaq.html">
                        <i class="fa fa-check"></i>
                        
                            <b>7.5.</b>
                        
                         Results FAQ
                    </a>
                
            
            
        </li>
    

            </ul>
            
        </li>
    
        
        <li class="chapter " data-level="8" data-path="references.html">
            
                
                    <a href="../references.html">
                        <i class="fa fa-check"></i>
                        
                            <b>8.</b>
                        
                         References
                    </a>
                
            
            
        </li>
    


        
        <li class="divider"></li>
        <li>
            <a href="http://www.gitbook.io/" target="blank" class="gitbook-link">Generated using GitBook</a>
        </li>
        
    </ul>
</div>

    <div class="book-body">
        <div class="body-inner">
            <div class="page-wrapper" tabindex="-1">
                <div class="book-progress">
    <div class="bar">
        <div class="inner" style="width: 3.3333333333333335%;min-width: 0%;"></div>
    </div>
    <div class="chapters">
    
        <a href="../index.html" title="Introduction" class="chapter done new-chapter" data-progress="0" style="left: 0%;"></a>
    
        <a href="../misc/Faq.html" title="FAQ" class="chapter done new-chapter" data-progress="1" style="left: 3.3333333333333335%;"></a>
    
        <a href="../misc/definitions.html" title="Definitions" class="chapter  new-chapter" data-progress="2" style="left: 6.666666666666667%;"></a>
    
        <a href="../misc/newsAndUpdates.html" title="News and Updates" class="chapter  new-chapter" data-progress="3" style="left: 10%;"></a>
    
        <a href="../installation/installation.html" title="Installation" class="chapter  new-chapter" data-progress="4" style="left: 13.333333333333334%;"></a>
    
        <a href="../installation/uninstallation.html" title="Uninstalling" class="chapter  new-chapter" data-progress="5" style="left: 16.666666666666668%;"></a>
    
        <a href="../installation/modules.html" title="Modules" class="chapter  new-chapter" data-progress="6" style="left: 20%;"></a>
    
        <a href="../dbcreator/dbcreator.html" title="Dbcreator module" class="chapter  " data-progress="6.1" style="left: 23.333333333333332%;"></a>
    
        <a href="../dbcreator/dbcreator_ucsc.html" title="dbcreator_ucsc" class="chapter  " data-progress="6.1.1" style="left: 26.666666666666668%;"></a>
    
        <a href="../dbcreator/dbcreator_encodeDCC.html" title="dbcreator_encodeDCC" class="chapter  " data-progress="6.1.2" style="left: 30%;"></a>
    
        <a href="../dbcreator/dbcreatorWorks.html" title="How dbcreator works" class="chapter  " data-progress="6.1.3" style="left: 33.333333333333336%;"></a>
    
        <a href="../dbcreator/dbcreatorFiltering.html" title="Filtering" class="chapter  " data-progress="6.1.4" style="left: 36.666666666666664%;"></a>
    
        <a href="../dbcreator/dbcreatorStructure.html" title="Database structure" class="chapter  " data-progress="6.1.5" style="left: 40%;"></a>
    
        <a href="../dbcreator/dbcreatorBackground.html" title="Background" class="chapter  " data-progress="6.1.6" style="left: 43.333333333333336%;"></a>
    
        <a href="../dbcreator/dbcreatorCustom.html" title="Custom data" class="chapter  " data-progress="6.1.7" style="left: 46.666666666666664%;"></a>
    
        <a href="../dbcreator/dbcreatorFAQ.html" title="Dbcreator FAQ" class="chapter  " data-progress="6.1.8" style="left: 50%;"></a>
    
        <a href="../optimizer/optimizer.html" title="Optimizer module" class="chapter  " data-progress="6.2" style="left: 53.333333333333336%;"></a>
    
        <a href="../optimizer/optimizerParallel.html" title="Parallelization" class="chapter  " data-progress="6.2.1" style="left: 56.666666666666664%;"></a>
    
        <a href="../optimizer/optimizerFAQ.html" title="Optimizer FAQ" class="chapter  " data-progress="6.2.2" style="left: 60%;"></a>
    
        <a href="../server/server.html" title="Server module" class="chapter  " data-progress="6.3" style="left: 63.333333333333336%;"></a>
    
        <a href="../server/serverParallel.html" title="Parallelization" class="chapter  " data-progress="6.3.1" style="left: 66.66666666666667%;"></a>
    
        <a href="../server/serverFAQ.html" title="Server FAQ" class="chapter  " data-progress="6.3.2" style="left: 70%;"></a>
    
        <a href="../hypergeom4/hypergeom4.html" title="Command line module" class="chapter  " data-progress="6.4" style="left: 73.33333333333333%;"></a>
    
        <a href="../hypergeom4/hypergeom4Faq.html" title="Command line FAQ" class="chapter  " data-progress="6.4.1" style="left: 76.66666666666667%;"></a>
    
        <a href="../hypergeom4/results.html" title="Results" class="chapter  new-chapter" data-progress="7" style="left: 80%;"></a>
    
        <a href="../hypergeom4/enrichment.html" title="Enrichment results" class="chapter  " data-progress="7.1" style="left: 83.33333333333333%;"></a>
    
        <a href="../hypergeom4/episimilarity.html" title="Epigenomic similarity results" class="chapter  " data-progress="7.2" style="left: 86.66666666666667%;"></a>
    
        <a href="../hypergeom4/annotation.html" title="Annotation results" class="chapter  " data-progress="7.3" style="left: 90%;"></a>
    
        <a href="../hypergeom4/logs.html" title="Logs and debug info" class="chapter  " data-progress="7.4" style="left: 93.33333333333333%;"></a>
    
        <a href="../hypergeom4/resultsFaq.html" title="Results FAQ" class="chapter  " data-progress="7.5" style="left: 96.66666666666667%;"></a>
    
        <a href="../references.html" title="References" class="chapter  new-chapter" data-progress="8" style="left: 100%;"></a>
    
    </div>
</div>

                <div class="page-inner">
                
                    <section class="normal" id="section-gitbook_19">
                    
                        <h1 id="faq">FAQ</h1>
<h3 id="-how-genomerunner-can-help-me">* How GenomeRunner can help me?</h3>
<p>GenomeRunner helps to interpret potential regulatory effect of SNPs (<a href="definitions.html">features of interest, FOIs</a>) by identifying functional elements (<a href="definitions.html">aka (epi)genomic features, GFs</a>) most statistically significantly co-localized with them (see <a href="definitions.html">Enrichment analysis</a>).</p>
<p>If one analyzes three or more sets of SNPs, such as SNPs from different individuals, populations, diseases, GenomeRunner visualizes their epigenomic similarity (see <a href="../hypergeom4/episimilarity.html">Epigenomic Similarity analysis</a>). This information may be used, e.g., to group patients by similarity of their individual sets of genomic variants within sell type-specific regulatory landscapes.</p>
<p><img src="../figures/GROverview.png" alt="GenomeRunner overview" title="GenomeRunner overview"></p>
<h3 id="-how-should-my-input-data-look-like-step-1">* How should my input data look like (Step 1)?</h3>
<p>Use tab-separated text files with genomic coordinates of the SNPs of interest in <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format1" target="_blank">BED format</a>, see <a href="http://uswest.ensembl.org/info/website/upload/bed.html" target="_blank">examples</a>. As a bare minimun, chromosome, start, and end coordinates should be provided. One can upload BED file(s), or copy-paste tab-separated coordinates.</p>
<p><strong>Note</strong>: a set of SNPs should contain at least 5 SNPs to be eligible for the analysis. Genomic coordinates should be 0-bases. The end coordinate should equal start coordinate + 1.</p>
<p><strong>Note</strong>: lists of new line-separated rsIDs (e.g., rs2789489\nrs4360154\nrs630642) can be submitted, provided a cross-mapping file has been created (see <a href="../dbcreator/dbcreatorCustom.html">Custom data</a>). This feature, however, should be considered experimental, use genomic coordinates where possible. Caveat emptor.</p>
<h3 id="-do-you-have-some-sets-of-snps-for-demo-runs">* Do you have some sets of SNPs for demo runs?</h3>
<p>Sure. Several buttons on the front page will select pre-defined sets of SNPs for the analysis.  For <em>Homo Sapiens</em> these include (To be updated):</p>
<table>
<thead>
<tr>
<th>Pre-defined sets of SNPs</th>
<th>When to use</th>
</tr>
</thead>
<tbody>
<tr>
<td><code>GWAS_vs_DGV</code> - selected disease-associated SNP sets from GWAScatalog.</td>
<td>Used with <em>gwasCatalog</em> background, to be run for enrichment vs. structural variants (dgvVariants)</td>
</tr>
<tr>
<td><code>GWAS_vs_tfbsEncode</code> - selected disease-associated SNP sets from GWAScatalog.</td>
<td>Used with <em>gwasCatalog</em> background, to be run for enrichment vs. 161 experimentally obtained transcription factor binding sites (tfbsEncode)</td>
</tr>
<tr>
<td><code>snp138Rand</code> - randomly selected sets of SNPs from the snp138 database.</td>
<td>Used with <em>snp138</em> background and any (category of) genome annotation elements, to ensure random SNPs do not show significant associations</td>
</tr>
<tr>
<td><code>gwasRand</code> - randomly selected sets of SNPs from  GWAScatalog.</td>
<td>Used with <code>gwascatalog</code> background and any (category of) genome annotation elements, to ensure random SNPs do not show significant associations</td>
</tr>
<tr>
<td><code>GWASmore15</code> - all disease- or trait-associated SNP sets from GWAScatalog having more than 15 SNPs in a set</td>
<td>Used with <code>gwascatalog</code> background for explorartory analyses vs. any (category of) genome annotation elements</td>
</tr>
</tbody>
</table>
<h3 id="-what-is-the-background-step-2">* What is the &quot;background&quot; (Step 2)?</h3>
<p>See the <a href="../dbcreator/dbcreatorBackground.html">Background</a> section. In short, the background is a “universe” of all SNPs assessed in a study, from which the SNPs of interest came from. Several pre-defined background sets are provided, for <em>Homo Sapiens</em> these include:</p>
<table>
<thead>
<tr>
<th>Pre-defined background</th>
<th>When to use</th>
</tr>
</thead>
<tbody>
<tr>
<td>snp138 (All Simple Nucleotide Polymorphisms (dbSNP 138))</td>
<td>For sets of SNPs from whole-genome GWA studies</td>
</tr>
<tr>
<td>snp138Common (Simple Nucleotide Polymorphisms (dbSNP 138) Found in &gt;= 1% of Samples)</td>
<td>For sets of SNPs from studies where rare variants were ignored</td>
</tr>
<tr>
<td>gwascatalog (NHGRI Catalog of Published Genome-Wide Association Studies)</td>
<td>For demo testing, to observe regulatory associations of disease-specific sets of SNPs, as compared with randomly selected SNPs from all GWAScatalog</td>
</tr>
</tbody>
</table>
<p>  For a GWAS, the background is likely to be all SNPs (snp138 for <em>Homo Sapiens</em>). For a study using microarrays, the background should contain coordinates of all SNPs on the microarray - upload or copy/paste them.</p>
<p><strong>Note</strong>: The SNPs of interest should be a subset of the background SNPs. If some SNPs of interest do not overlap the background, a non-critical error is issued. Use <a href="https://github.com/arq5x/bedtools2" target="_blank">BEDtools</a> for creating custom backgrounds and for any other manipulations with the BED files.</p>
<h3 id="-what-are-genome-annotation-features-step-3">* What are &quot;genome annotation features&quot; (Step 3)?</h3>
<p><a href="definitions.html">Genome annotations</a> are discrete regions potentially having functional/regulatory properties. Vast majority of them were experimentally obtained by the <a href="http://www.genome.gov/encode/" target="_blank">ENCODE project</a>.</p>
<h3 id="-there-are-just-too-many-genome-annotation-features-what-to-choose">* There are just too many genome annotation features!!! What to choose?</h3>
<p>Don&#39;t panic. The genome annotation features are organized by categories mirrored from the UCSC genome browser (see <a href="../dbcreator/dbcreatorStructure.html">Database structure</a>). Use search box and/or checkboxes in the TreeView control to select one or more genome annotation categories. Clicking on a genome annotation’ name will bring up description, if available.</p>
<p>The ENCODE data are organized by source/data type, tiers (quality), and by cell types. <em>Hint</em>: Several well-known/specially processes genome annotation features sets are brought forward as “default genome annotation features”. For <em>Homo Sapiens</em> these include:</p>
<table>
<thead>
<tr>
<th>Genome annotation category</th>
<th>Experimental question: Are the SNPs of interest...</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=knownAlt&amp;hgTracksConfigPage=configure" target="_blank">altSplicing</a> (Alternative Splicing, Alternative Promoter and Similar Events in UCSC Genes, split by splicing type)</td>
<td>... potentially disrupt a specific type of alternative spliced regions?</td>
</tr>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=wgEncodeBroadHmm&amp;hgTracksConfigPage=configure" target="_blank">chromStates</a> (Chromatin State Segmentation by HMM from ENCODE/Broad, Gm12878 cell line, split by chromatin state type)</td>
<td>... preferentially located in certain chromatin states?</td>
</tr>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=coriellDelDup&amp;hgTracksConfigPage=configure" target="_blank">coriellVariants</a> (Coriell Cell Line Copy Number Variants, split by cell types)</td>
<td>... enriched in CNVs, and in which cell type?</td>
</tr>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=gap&amp;hgTracksConfigPage=configure" target="_blank">gapLocations</a> (Gap locations)</td>
<td>... happen to be in gaps, telomeres, heterochromatin regions?</td>
</tr>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=dgvPlus&amp;hgTracksConfigPage=configure" target="_blank">genomicVariants</a> (Database of Genomic Variants: Structural Variation (CNV, Inversion, In/del), split by variant type)</td>
<td>... enriched in CNVs, or other types of structural variations?</td>
</tr>
<tr>
<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/23263488" target="_blank">H3K4me3</a> (Tissue-specific histone 3 lysine 4 trimethylation marks)</td>
<td>... enriched in tissue-specific active transcription-associated histone mark?</td>
</tr>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=wgEncodeBroadHistone&amp;hgTracksConfigPage=configure" target="_blank">histoneMarks</a> (Histone modifications by ChIP-seq from ENCODE/Broad institute, Gm12878 cell line</td>
<td>... enriched in B-lymphoblastoma-associated histone modification marks?</td>
</tr>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=wgRna&amp;hgTracksConfigPage=configure" target="_blank">ncRNAs</a> (C/D and H/ACA Box snoRNAs, scaRNAs, and microRNAs from snoRNABase and miRBase, split by ncRNA type)</td>
<td>... associated with a class of non-coding elements?</td>
</tr>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=rmsk&amp;hgTracksConfigPage=configure" target="_blank">repeats</a> (Repeating Elements by RepeatMasker, split by repeat class)</td>
<td>... enriched in regions of low complexity, and in which type?</td>
</tr>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=tfbsConsSites&amp;hgTracksConfigPage=configure" target="_blank">tfbsConserved</a> (HMR Conserved Transcription Factor Binding Sites, split by TFBS name)</td>
<td>... potentially disrupt a specific computationally predicted transcription factor binding site?</td>
</tr>
<tr>
<td><a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&amp;g=wgEncodeRegTfbsClusteredV2&amp;hgTracksConfigPage=configure" target="_blank">tfbsEncode</a> (Transcription Factor ChIP-seq Clusters V3 (161 targets, 189 antibodies) from ENCODE, split by TFBS name)</td>
<td>... potentially disrupt a specific experimentally defined transcription factor binding site?</td>
</tr>
</tbody>
</table>
<!--| [gwasCatalog](https://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=381293375_xxuzKzx80QrJD5K5yoLpCkSoHtwk&g=gwasCatalog&hgTracksConfigPage=configure) (NHGRI Catalog of Published Genome-Wide Association Studies, split by disease/trait types)                | ... enriched in known disease-specific SNPs?                                                    |-->

<p>Examples of what to choose:</p>
<ul>
<li>Select the <code>tfbsEncode</code> category to get an answer whether the SNPs of interest are enriched in any of the 161 transcription factor binding sites identified by ChIP-seq.</li>
<li>Select the <code>histoneMarks</code> category to get an insight whether the SNPs of interest are enriched in histone marks in B lymphocytes.</li>
<li>Select the <code>genes</code> category to answer a question whether the SNPs of interest are enriched in genes/exons.</li>
</ul>
<h3 id="-how-genomerunner-web-server-is-different-from-previously-published-version">* How GenomeRunner web server is different from previously published version?</h3>
<p>The original version of GenomeRunner, hosted on <a href="http://sourceforge.net/projects/genomerunner/" target="_blank">SourceForge</a>, was designed as an &quot;all purpose&quot; tool. It has several advantages over the web interface, as well as disadvantages, such as learning curve, various non-obvious settings, need to download large databases, restriction to Windows platform, lacking visualization capabilities. GenomeRunner web server addresses these issues - it is:</p>
<ul>
<li>Platform-independent, runs in a browser (Google Chrome recommended)</li>
<li>Simple and intuitive interface</li>
<li>Visual, the results are presented as interactive heatmaps</li>
<li>Implements a novel idea of <a href="../hypergeom4/episimilarity.html">epigenomic similarity analysis</a>, to understand relationships among sets of SNPs on regulatory level</li>
<li>Downloads-free, we host and maintain genome annotation databases</li>
<li>Supports local installation and command line usage</li>
</ul>
<h3 id="-how-to-cite-genomerunner">* How to cite GenomeRunner?</h3>
<p>If you find GenomeRunner useful, please, <a href="../references.html">cite the 2012 paper</a>. The manuscript describing the web version is in preparation</p>
<h3 id="-i-still-have-questionssuggestionsbug-report-what-to-do">* I still have questions/suggestions/bug report. What to do?</h3>
<p>Please, contact <img src="../figures/e-mail.png" alt="Mikhail Dozmorov at gmail" title="Mikhail Dozmorov"></p>

                    
                    </section>
                
                </div>
            </div>
        </div>

        
        <a href="../index.html" class="navigation navigation-prev " aria-label="Previous page: Introduction"><i class="fa fa-angle-left"></i></a>
        
        
        <a href="../misc/definitions.html" class="navigation navigation-next " aria-label="Next page: Definitions"><i class="fa fa-angle-right"></i></a>
        
    </div>
</div>

        
<script src="../gitbook/jsrepl/jsrepl.js" id="jsrepl-script"></script>
<script src="../gitbook/app.js"></script>

    
    <script src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    

    
    <script src="../gitbook/plugins/gitbook-plugin-mathjax/plugin.js"></script>
    

<script>
require(["gitbook"], function(gitbook) {
    var config = {};
    gitbook.start(config);
});
</script>

        
    </body>
    
</html>
