<!DOCTYPE HTML>
<html lang="en-US" manifest="../manifest.appcache">
    <head prefix="og: http://ogp.me/ns# book: http://ogp.me/ns/book#">
        
        <meta charset="UTF-8">
        <title>FAQ | GenomeRunner</title>

        <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
        <meta name="robots" content="index, follow">
        <meta name="author" content="mdozmorov">
        <meta name="description" content="GenomeRunner Web is a tool for functional interpretation of sets of single nucleotide polymorphisms (SNPs) by considering their co-localization with functional/regulatory genome annotation data (epigenomic elements). It is particularly useful for interpretation of functional roles of rare variants and SNPs in non-protein coding regions. An example of GenomeRunner&#39;s results can be found in the analysis of Sjogren&#39;s syndrome GWAS (Nature Genetics), where it identified RFX5 transcription factor binding site as the most statistically significantly co-localized with the disease-associated SNPs.">
        <meta name="keywords" content="gitbook,github" >
        <meta name="generator" content="www.gitbook.io">

        
        <link rel="next" href="../misc/definitions.html" />
        
        
        <link rel="prev" href="../index.html" />
        

        <meta property="og:title" content="FAQ | GenomeRunner">
        <meta property="og:site_name" content="GenomeRunner">
        <meta property="og:type" content="book">
        <meta property="og:locale" content="en_US">

        <meta property="book:author" content="https://github.com/mdozmorov">
        <meta property="book:tag" content="GitBook">

        <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
        <meta name="apple-mobile-web-app-capable" content="yes">
        <meta name="apple-mobile-web-app-status-bar-style" content="black">

        <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">
        
    </head>
    <body>
        

        <link rel="stylesheet" href="../gitbook/style.css">
        


        
    <div class="book" data-github="mdozmorov/grdocs" data-level="1" data-basepath=".." data-revision="1402282482484">
    <div class="book-header">
    <!-- Actions Left -->
    
    <a href="https://github.com/mdozmorov/grdocs" target="_blank" class="btn pull-left home-bookmark" aria-label="GitHub home"><i class="fa fa-bookmark-o"></i></a>
    
    <a href="#" class="btn pull-left toggle-summary" aria-label="Toggle summary"><i class="fa fa-align-justify"></i></a>
    <a href="#" class="btn pull-left toggle-search" aria-label="Toggle search"><i class="fa fa-search"></i></a>
    <span id="font-settings-wrapper">
        <a href="#" class="btn pull-left toggle-font-settings" aria-label="Toggle font settings"><i class="fa fa-font"></i>
        </a>
        <div class="dropdown-menu font-settings">
    <div class="dropdown-caret">
        <span class="caret-outer"></span>
        <span class="caret-inner"></span>
    </div>

    <div class="btn-group btn-block">
        <button id="reduce-font-size" class="btn btn-default">A</button>
        <button id="enlarge-font-size" class="btn btn-default">A</button>
    </div>

    <ul class="list-group font-family-list">
        <li class="list-group-item" data-font="0">Serif</li>
        <li class="list-group-item" data-font="1">Sans</li>
    </ul>

    <div class="btn-group btn-group-xs btn-block color-theme-list">
        <button type="button" class="btn btn-default" id="color-theme-preview-0" data-theme="0">White</button>
        <button type="button" class="btn btn-default" id="color-theme-preview-1" data-theme="1">Sepia</button>
        <button type="button" class="btn btn-default" id="color-theme-preview-2" data-theme="2">Night</button>
    </div>
</div>

    </span>

    <!-- Actions Right -->
    
    <a href="#" target="_blank" class="btn pull-right google-plus-sharing-link sharing-link" data-sharing="google-plus" aria-label="Share on Google Plus"><i class="fa fa-google-plus"></i></a>
    
    
    <a href="#" target="_blank" class="btn pull-right facebook-sharing-link sharing-link" data-sharing="facebook" aria-label="Share on Facebook"><i class="fa fa-facebook"></i></a>
    
    
    <a href="#" target="_blank" class="btn pull-right twitter-sharing-link sharing-link" data-sharing="twitter" aria-label="Share on Twitter"><i class="fa fa-twitter"></i></a>
    

    
    <a href="https://github.com/mdozmorov/grdocs/stargazers" target="_blank" class="btn pull-right count-star hidden-xs"><i class="fa fa-star-o"></i> Star (<span>-</span>)</a>
    <a href="https://github.com/mdozmorov/grdocs/watchers" target="_blank" class="btn pull-right count-watch hidden-xs"><i class="fa fa-eye"></i> Watch (<span>-</span>)</a>
    

    <!-- Title -->
    <h1>
        <i class="fa fa-spinner fa-spin"></i>
        <a href="../" >GenomeRunner</a>
    </h1>
</div>

    <div class="book-summary">
    <div class="book-search">
        <input type="text" placeholder="Search" class="form-control" />
    </div>
    <ul class="summary">
        
        <li>
            <a href="https://github.com/mdozmorov" target="blank" class="author-link">About the author</a>
        </li>
        

        
        <li>
            <a href="https://github.com/mdozmorov/grdocs/issues" target="blank"class="issues-link">Questions and Issues</a>
        </li>
        

        
        <li>
            <a href="https://github.com/mdozmorov/grdocs/edit/master/misc/Faq.md" target="blank" class="contribute-link">Edit and Contribute</a>
        </li>
        

        

        <li data-level="0" data-path="index.html">
            <a href="../index.html"><i class="fa fa-check"></i> Introduction</a>
        </li>
        
            <li class="chapter " data-level="1" data-path="misc/Faq.html">
                
                <a href="../misc/Faq.html">
                    <i class="fa fa-check"></i> <b>1.</b> FAQ
                </a>
                
                
            </li>
        
            <li class="chapter " data-level="2" data-path="misc/definitions.html">
                
                <a href="../misc/definitions.html">
                    <i class="fa fa-check"></i> <b>2.</b> Definitions
                </a>
                
                
            </li>
        
            <li class="chapter " data-level="3" data-path="misc/newsAndUpdates.html">
                
                <a href="../misc/newsAndUpdates.html">
                    <i class="fa fa-check"></i> <b>3.</b> News and Updates
                </a>
                
                
            </li>
        
            <li class="chapter " data-level="4" data-path="installation/installation.html">
                
                <a href="../installation/installation.html">
                    <i class="fa fa-check"></i> <b>4.</b> Installation
                </a>
                
                
            </li>
        
            <li class="chapter " data-level="5" data-path="installation/uninstallation.html">
                
                <a href="../installation/uninstallation.html">
                    <i class="fa fa-check"></i> <b>5.</b> Uninstalling
                </a>
                
                
                <ul class="articles">
                    
                        <li  data-level="5.1" data-path="installation/modules.html">
                            
                            <a href="../installation/modules.html">
                                <i class="fa fa-check"></i> <b>5.1.</b> Modules
                            </a>
                            
                        </li>
                    
                </ul>
                
            </li>
        
            <li class="chapter " data-level="6" data-path="dbcreator/dbcreator.html">
                
                <a href="../dbcreator/dbcreator.html">
                    <i class="fa fa-check"></i> <b>6.</b> Dbcreator module
                </a>
                
                
                <ul class="articles">
                    
                        <li  data-level="6.1" data-path="dbcreator/dbcreatorWorks.html">
                            
                            <a href="../dbcreator/dbcreatorWorks.html">
                                <i class="fa fa-check"></i> <b>6.1.</b> How dbcreator works
                            </a>
                            
                        </li>
                    
                        <li  data-level="6.2" data-path="dbcreator/dbcreatorFiltering.html">
                            
                            <a href="../dbcreator/dbcreatorFiltering.html">
                                <i class="fa fa-check"></i> <b>6.2.</b> Filtering
                            </a>
                            
                        </li>
                    
                        <li  data-level="6.3" data-path="dbcreator/dbcreatorStructure.html">
                            
                            <a href="../dbcreator/dbcreatorStructure.html">
                                <i class="fa fa-check"></i> <b>6.3.</b> Database structure
                            </a>
                            
                        </li>
                    
                        <li  data-level="6.4" data-path="dbcreator/dbcreatorBackground.html">
                            
                            <a href="../dbcreator/dbcreatorBackground.html">
                                <i class="fa fa-check"></i> <b>6.4.</b> Background
                            </a>
                            
                        </li>
                    
                        <li  data-level="6.5" data-path="dbcreator/dbcreatorCustom.html">
                            
                            <a href="../dbcreator/dbcreatorCustom.html">
                                <i class="fa fa-check"></i> <b>6.5.</b> Custom data
                            </a>
                            
                        </li>
                    
                        <li  data-level="6.6" data-path="dbcreator/dbcreatorFAQ.html">
                            
                            <a href="../dbcreator/dbcreatorFAQ.html">
                                <i class="fa fa-check"></i> <b>6.6.</b> Dbcreator FAQ
                            </a>
                            
                        </li>
                    
                </ul>
                
            </li>
        
            <li class="chapter " data-level="7" data-path="optimizer/optimizer.html">
                
                <a href="../optimizer/optimizer.html">
                    <i class="fa fa-check"></i> <b>7.</b> Optimizer module
                </a>
                
                
                <ul class="articles">
                    
                        <li  data-level="7.1" data-path="optimizer/optimizerFAQ.html">
                            
                            <a href="../optimizer/optimizerFAQ.html">
                                <i class="fa fa-check"></i> <b>7.1.</b> Optimizer FAQ
                            </a>
                            
                        </li>
                    
                </ul>
                
            </li>
        
            <li class="chapter " data-level="8" data-path="server/server.html">
                
                <a href="../server/server.html">
                    <i class="fa fa-check"></i> <b>8.</b> Server module
                </a>
                
                
                <ul class="articles">
                    
                        <li  data-level="8.1" data-path="server/serverFAQ.html">
                            
                            <a href="../server/serverFAQ.html">
                                <i class="fa fa-check"></i> <b>8.1.</b> Server FAQ
                            </a>
                            
                        </li>
                    
                </ul>
                
            </li>
        
            <li class="chapter " data-level="9" data-path="hypergeom4/hypergeom4.html">
                
                <a href="../hypergeom4/hypergeom4.html">
                    <i class="fa fa-check"></i> <b>9.</b> Command line module
                </a>
                
                
                <ul class="articles">
                    
                        <li  data-level="9.1" data-path="hypergeom4/results.html">
                            
                            <a href="../hypergeom4/results.html">
                                <i class="fa fa-check"></i> <b>9.1.</b> Results
                            </a>
                            
                        </li>
                    
                        <li  data-level="9.2" data-path="hypergeom4/hypergeom4Faq.html">
                            
                            <a href="../hypergeom4/hypergeom4Faq.html">
                                <i class="fa fa-check"></i> <b>9.2.</b> Command line FAQ
                            </a>
                            
                        </li>
                    
                </ul>
                
            </li>
        
            <li class="chapter " data-level="10" data-path="references.html">
                
                <a href="../references.html">
                    <i class="fa fa-check"></i> <b>10.</b> References
                </a>
                
                
            </li>
        

        
        <li class="divider"></li>
        <li>
            <a href="http://www.gitbook.io/" target="blank" class="gitbook-link">Generated using GitBook</a>
        </li>
        
    </ul>
</div>

    <div class="book-body">
        <div class="body-inner">
            <div class="page-wrapper" tabindex="-1">
                <div class="book-progress">
    <div class="bar">
        <div class="inner" style="width: 4.761904761904762%;min-width: 0%;"></div>
    </div>
    <div class="chapters">
    
        <a href="../index.html" title="Introduction" class="chapter done new-chapter" data-progress="0" style="left: 0%;"></a>
    
        <a href="../misc/Faq.html" title="FAQ" class="chapter done new-chapter" data-progress="1" style="left: 4.761904761904762%;"></a>
    
        <a href="../references.html" title="References" class="chapter  " data-progress="10" style="left: 9.523809523809524%;"></a>
    
        <a href="../misc/definitions.html" title="Definitions" class="chapter  new-chapter" data-progress="2" style="left: 14.285714285714286%;"></a>
    
        <a href="../misc/newsAndUpdates.html" title="News and Updates" class="chapter  new-chapter" data-progress="3" style="left: 19.047619047619047%;"></a>
    
        <a href="../installation/installation.html" title="Installation" class="chapter  new-chapter" data-progress="4" style="left: 23.80952380952381%;"></a>
    
        <a href="../installation/uninstallation.html" title="Uninstalling" class="chapter  new-chapter" data-progress="5" style="left: 28.571428571428573%;"></a>
    
        <a href="../installation/modules.html" title="Modules" class="chapter  " data-progress="5.1" style="left: 33.333333333333336%;"></a>
    
        <a href="../dbcreator/dbcreator.html" title="Dbcreator module" class="chapter  new-chapter" data-progress="6" style="left: 38.095238095238095%;"></a>
    
        <a href="../dbcreator/dbcreatorWorks.html" title="How dbcreator works" class="chapter  " data-progress="6.1" style="left: 42.857142857142854%;"></a>
    
        <a href="../dbcreator/dbcreatorFiltering.html" title="Filtering" class="chapter  " data-progress="6.2" style="left: 47.61904761904762%;"></a>
    
        <a href="../dbcreator/dbcreatorStructure.html" title="Database structure" class="chapter  " data-progress="6.3" style="left: 52.38095238095238%;"></a>
    
        <a href="../dbcreator/dbcreatorBackground.html" title="Background" class="chapter  " data-progress="6.4" style="left: 57.142857142857146%;"></a>
    
        <a href="../dbcreator/dbcreatorCustom.html" title="Custom data" class="chapter  " data-progress="6.5" style="left: 61.904761904761905%;"></a>
    
        <a href="../dbcreator/dbcreatorFAQ.html" title="Dbcreator FAQ" class="chapter  " data-progress="6.6" style="left: 66.66666666666667%;"></a>
    
        <a href="../optimizer/optimizer.html" title="Optimizer module" class="chapter  new-chapter" data-progress="7" style="left: 71.42857142857143%;"></a>
    
        <a href="../optimizer/optimizerFAQ.html" title="Optimizer FAQ" class="chapter  " data-progress="7.1" style="left: 76.19047619047619%;"></a>
    
        <a href="../server/server.html" title="Server module" class="chapter  new-chapter" data-progress="8" style="left: 80.95238095238095%;"></a>
    
        <a href="../server/serverFAQ.html" title="Server FAQ" class="chapter  " data-progress="8.1" style="left: 85.71428571428571%;"></a>
    
        <a href="../hypergeom4/hypergeom4.html" title="Command line module" class="chapter  new-chapter" data-progress="9" style="left: 90.47619047619048%;"></a>
    
        <a href="../hypergeom4/results.html" title="Results" class="chapter  " data-progress="9.1" style="left: 95.23809523809524%;"></a>
    
        <a href="../hypergeom4/hypergeom4Faq.html" title="Command line FAQ" class="chapter  " data-progress="9.2" style="left: 100%;"></a>
    
    </div>
</div>

                <div class="page-inner">
                
                    <section class="normal" id="section-gitbook_11">
                    
                        <h1 id="faq">FAQ</h1>
<ul>
<li>How GenomeRunner can help me?</li>
</ul>
<blockquote>
<p>GenomeRunner helps to interpret regulatory effect of SNPs by identifying functional elements (<a href="definitions.html">genomic features, GFs</a>) most statistically significantly co-localized with them (see <a href="definitions.html">Enrichment analysis</a>). These regulatory elements may provide understanding which mechanisms (and in which cell type) may be affected by the SNPs of interest (<a href="definitions.html">features of interest, FOIs</a>).</p>
<p>If one analyzes three or more sets of SNPs, such as SNPs from different individuals, populations, diseases, GenomeRunner visualizes their epigenomic similarity (see <a href="definitions.html">Epigenomic Similarity analysis</a>). This information may be used to classify the sets of SNPs by similarity of their effect upon regulatory landscape.</p>
</blockquote>
<ul>
<li>How my input data should look like (Step 1)?</li>
</ul>
<blockquote>
<p>Tab-separated text files with genomic coordinates of the SNPs of interest in <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format1" target="_blank">BED format</a>. As a bare minimun, chromosome, start, and end coordinates should be provided. One can upload BED file(s), or copy-paste tab-separated coordinates.</p>
<p><strong>Note</strong>: a set of SNPs should contain at least 5 SNPs to be eligible for the analysis. Genomic coordinates should be 0-bases. The end coordinate should equal start coordinate + 1.</p>
</blockquote>
<ul>
<li>Do you have any examples?</li>
</ul>
<blockquote>
<p>Sure. Several buttons on the front page will select pre-defined sets of SNPs for the analysis.  For <em>Homo Sapiens</em> these include:</p>
</blockquote>
<table>
<thead>
<tr>
<th>Pre-defined sets of SNPs</th>
<th>When to use</th>
</tr>
</thead>
<tbody>
<tr>
<td>GWAS_vs_DGV - selected disease-associated SNP sets from GWAScatalog.</td>
<td>Used for Demo run, to be run for enrichment vs. structural variants (dgvVariants)</td>
</tr>
<tr>
<td>GWAS_vs_tfbsEncode - selected disease-associated SNP sets from GWAScatalog.</td>
<td>Used for Demo run, to be run for enrichment vs. 161 experimentally obtained transcription factor binding sites (tfbsEncode)</td>
</tr>
<tr>
<td>snp138Rand - randomly selected sets of SNPs from the snp138 database.</td>
<td>Use with snp138+ background and any (category of) genome annotation elements, to ensure random SNPs do not show significant associations</td>
</tr>
<tr>
<td>gwasRand - randomly selected sets of SNPs from  GWAScatalog.</td>
<td>Use with gwascatalog background and any (category of) genome annotation elements, to ensure random SNPs do not show significant associations</td>
</tr>
<tr>
<td>GWASmore15 - all disease- or trait-associated SNP sets from GWAScatalog having more than 15 SNPs in a set</td>
<td>Used for explorartory analyses vs. any (category of) genome annotation elements</td>
</tr>
</tbody>
</table>
<ul>
<li>What is &quot;background&quot; (Step 2)?</li>
</ul>
<blockquote>
<p>See <a href="../dbcreator/dbcreatorBackground.html">Background</a> section. In short, the background is the “universe” of all SNPs assessed in a study, from which the SNPs of interest came from. Several pre-defined backgrounds are provided,  for <em>Homo Sapiens</em> these include:</p>
</blockquote>
<table>
<thead>
<tr>
<th>Pre-defined background</th>
<th>When to use</th>
</tr>
</thead>
<tbody>
<tr>
<td>snp138+ (All Simple Nucleotide Polymorphisms (dbSNP 138))</td>
<td>For SNPs from whole-genome GWA studies</td>
</tr>
<tr>
<td>snp138Common+ (Simple Nucleotide Polymorphisms (dbSNP 138) Found in &gt;= 1% of Samples)</td>
<td>For SNPs from studies where rare variants were ignored</td>
</tr>
<tr>
<td>gwascatalog+ (NHGRI Catalog of Published Genome-Wide Association Studies)</td>
<td>For demo testing, to observe regulatory associations of disease-specific sets of SNPs, as compared with randomly selected SNPs from all GWAScatalog</td>
</tr>
</tbody>
</table>
<blockquote>
<p>For a GWAS, the background is likely to be all SNPs (snp138+). For a study using microarrays, the background would contain coordinates of all SNPs on the microarray - upload or copy/paste them.</p>
<p><strong>Note</strong>: The SNPs of interest should be a subset of the background SNPs. If some SNPs of interest do not overlap the background, a non-critical error is issued. Create a custom background that includes all SNPs of interest. Use <a href="https://github.com/arq5x/bedtools2" target="_blank">BEDtools</a> for any manipulations with BED files.</p>
</blockquote>
<ul>
<li>What are &quot;genome annotation features&quot; (Step 3)?</li>
</ul>
<blockquote>
<p>Genome annotations are discrete regions potentially having functional/regulatory properties. See <a href="definitions.html">Definitions</a> section.</p>
</blockquote>
<ul>
<li>There are too many genome annotation features! What to choose?</li>
</ul>
<blockquote>
<p>The genome annotation features are organized by categories mirrored from the UCSC genome browser. Using search box and/or checkboxes in the TreeView control, select one or more genome annotation categories. Clicking on a genome annotation’ name will bring up description, if available.</p>
<p>The ENCODE data are organized by the source/data type, tiers (quality), and by cell types. Hint: Several well-known/specially processes genome annotation features sets are brought forward as “default genome annotation features”. For <em>Homo Sapiens</em> these include:</p>
</blockquote>
<table>
<thead>
<tr>
<th>Genome annotation category</th>
<th>Experimental question: Are the SNPs of interest...</th>
</tr>
</thead>
<tbody>
<tr>
<td>coriellVariants (Coriell Cell Line Copy Number Variants, split by cell types)</td>
<td>... enriched in CNVs, and in which cell type?</td>
</tr>
<tr>
<td>dgvVariants (Database of Genomic Variants: Structural Variation (CNV, Inversion, In/del), split by variant type)</td>
<td>... enriched in CNVs, or other types of structural variations?</td>
</tr>
<tr>
<td>repeats (Repeating Elements by RepeatMasker, split by repeat class)</td>
<td>... enriched in regions of low complexity, and in which type?</td>
</tr>
<tr>
<td>gwasCatalog (NHGRI Catalog of Published Genome-Wide Association Studies, split by disease/trait types)</td>
<td>... enriched in known disease-specific SNPs?</td>
</tr>
<tr>
<td>altSplicing (Alternative Splicing, Alternative Promoter and Similar Events in UCSC Genes, split by splicing type)</td>
<td>... potentially disrupt a specific type of alternative spliced regions?</td>
</tr>
<tr>
<td>ncRNAs (C/D and H/ACA Box snoRNAs, scaRNAs, and microRNAs from snoRNABase and miRBase, split by ncRNA type)</td>
<td>... associated with a class of non-coding elements?</td>
</tr>
<tr>
<td>tfbsEncode (Transcription Factor ChIP-seq Clusters V3 (161 targets, 189 antibodies) from ENCODE, split by TFBS name)</td>
<td>... potentially disrupt a specific experimentally defined transcription factor binding site?</td>
</tr>
<tr>
<td>tfbsConserved (HMR Conserved Transcription Factor Binding Sites, split by TFBS name)</td>
<td>... potentially disrupt a specific computationally predicted transcription factor binding site?</td>
</tr>
<tr>
<td>chromStates (Chromatin State Segmentation by HMM from ENCODE/Broad, Gm12878 cell line, split by chromatin state type)</td>
<td>... preferentially located in certain chromatin states?</td>
</tr>
</tbody>
</table>
<blockquote>
<p>Examples of what to choose:</p>
<ul>
<li>Select “tfbsEncode” category to get an answer whether the SNPs of interest are enriched in any of the 161 transcription factor binding sites identified by ChIP-seq.</li>
<li>Select “ENCODE/BroadHistone/Tier1/Gm12878” category to get an insight whether the SNPs of interest are enriched in histone marks in B lymphocytes.</li>
<li>Select “genes” category to answer a question, whether the SNPs of interest are enriched in genes/exons.</li>
</ul>
</blockquote>
<ul>
<li>How GenomeRunner web server is different from previously published version?</li>
</ul>
<blockquote>
<p>The original version of GenomeRunner, hosted on <a href="http://sourceforge.net/projects/genomerunner/" target="_blank">SourceForge</a>, was designed as &quot;all purpose&quot; tool. It has several disadvantages, such as learning curve among all the settings, download of large databases, restriction to Windows platform, lacking visualization capabilities. GenomeRunner web server addresses these issues - it is:</p>
<ul>
<li>Platform-independent, runs in a browser (Google Chrome recommended)</li>
<li>Simple and intuitive interface</li>
<li>Visual, the results are presented as interactive heatmaps</li>
<li>Implements a novel idea of <a href="definitions.html">epigenomic similarity analysis</a>, to understand relationships among the sets of SNPs on regulatory level</li>
<li>Downloads-free, we host and maintain genome annotation databases</li>
<li>Supports local installation and command line usage</li>
</ul>
</blockquote>
<ul>
<li>How to cite GenomeRunner?</li>
</ul>
<blockquote>
<p>If you find GenomeRunner useful, please, <a href="../references.html">cite the 2012 paper</a>.</p>
</blockquote>
<ul>
<li>I have questions/suggestions/bug report. What to do?</li>
</ul>
<blockquote>
<p>Please, contact <img src="../figures/e-mail.png" alt="Mikhail Dozmorov at gmail" title="Mikhail Dozmorov"></p>
</blockquote>

                    
                    </section>
                
                </div>
            </div>
        </div>

        
        <a href="../index.html" class="navigation navigation-prev " aria-label="Previous page: Introduction"><i class="fa fa-angle-left"></i></a>
        
        
        <a href="../misc/definitions.html" class="navigation navigation-next " aria-label="Next page: Definitions"><i class="fa fa-angle-right"></i></a>
        
    </div>
</div>

        

        <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.1.3/ace.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.1.3/mode-javascript.js"></script>
        <script src="../gitbook/jsrepl/jsrepl.js" id="jsrepl-script"></script>
        <script src="../gitbook/app.js"></script>
        

    
    <script src="../gitbook/plugins/gitbook-plugin-mixpanel/plugin.js"></script>
    

    
    <script src="http://cdn.mathjax.org/mathjax/2.0-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    

    
    <script src="../gitbook/plugins/gitbook-plugin-mathjax/plugin.js"></script>
    

<script>
require(["gitbook"], function(gitbook) {
    var config = {};
    gitbook.start(config);
});
</script>

    </body>
</html>
